usagov/usagov-2021
# USAGOV-1853-allow-wizard-deselect-on-spacebar: Allow users to deselect wizard radios by hitting spacebar.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1853

## Description

Create a script called wizardChecked.js (this was prevoiusly deleted, and now it's needed again for a different reason) and add it to the wizardStepTaxonomy library in the librariies.yml.

## Type of Changes

- [ X ] New Feature
- [ X ] Frontend (Twig, Sass, JS)

## Testing Instructions


1. Load any wizard page and open up devtools console.
2. There are no console errors. 
3. Tab into the first wizard radio option.
4. You see visual focus around the radio button.
5. Click spacebar.
6. The option becomes selected.
7. Click spacebar again.
8. The option becomes deselected.
9. Hit the "next" button.
10. The error message appears as expected.
11. Hit shift+tab until you are tabbed back into the radio list.
12. Navigate through the radio list with the left and right arrows on the keyboard.
13. Select one of the options by hitting spacebar.
14. Hit next.
15. You are taken to the next page as expected.
16. Hit the previous button.
17. The previous button behaves as expected.
18. On the next page or the previous page, click on an option.
19. Hit the next Button.
20. The next button behaves as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
21. Find the commit of this pull request.
22. Build and deploy the changes.
23. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
